googleapiclient/http.py

```diff
@@ -38,11 +38,34 @@
 import os
 import random
 import socket
-import ssl
 import sys
 import time
 import uuid
 
+
+try:
+  import ssl
+except ImportError:
+  class _ssl_SSLError(socket.error):
```

Comment:
I don't see why this class is necessary - it's never instantiated, so the properties defined within it can never be accessed, right? The problem seems to be that _if_ the `ssl` module is available, we want to be able to catch instances of `ssl.SSLError`, but that if the `ssl` module is not available, _we only want the code not to crash_. Right?

Why isn't the code that you're adding to the imports of this module just
```
try:
  import ssl
except ImportError:
  _ssl_SSLError = object()
else:
  _ssl_SSLError = ssl.SSLError
```
?